e2e/cypress/tests/integration/enterprise/guest_accounts/guest_invitation_ui_spec.ts

```diff
@@ -223,4 +223,15 @@ describe('Guest Account - Guest User Invitation Flow', () => {
         // * Verify invite more button is present
         cy.findByTestId('invite-more').should('be.visible');
     });
+
+    it('hides the copy link button when inviting guests', () => {
+        // # Open team menu and click 'Invite People'
+        cy.uiOpenTeamMenu('Invite People');
+
+        // # Select Guest
```

Comment:
nit: `// * Select Guest`

Reply:
For me, it's more an action than an assertion. By that I mean that I value the `click()` much more than I value the `should(...)`